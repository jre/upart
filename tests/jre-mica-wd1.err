upart: extended MBR in sector 125850535 has invalid magic number
