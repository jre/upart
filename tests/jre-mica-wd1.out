jre-mica-wd1.img: 149GB (312581808 sectors of 512 bytes)

MBR partition table at sector 0 of jre-mica-wd1.img:
                 Start            Size A Type
0:               16065        83891430 * Solaris (0xbf)
 Sun x86 disk label at sector 16065 (offset 1) of jre-mica-wd1.img:
                  Start            Size Flags Type
 0:            42588315        41287050 wm    root
 1:               64260         1237005 wu    swap
 2:               16065        83859300 wm    backup
 7:             1301265        41287050 wm    home
 8:               16065           16065 wu    boot
 9:               32130           32130 wu    altsctr
1:            83907495        20971520   OpenBSD (0xa6)
2:           104879015        20971520   FreeBSD or DragonFly (0xa5)
3:           125850535       186731273   Windows Extended (LBA) (0x0f)
